package.json

```
@@ -141,7 +141,7 @@
         "@react-stately/toggle": "3.4.2",
         "@react-stately/tooltip": "3.2.2",
         "@react-stately/tree": "3.3.4",
-        "@udecode/plate": "18.9.0",
+        "@udecode/plate": "^18.11.2",
```

```suggestion
        "@udecode/plate": "18.11.2",
```
And then `npm i` again to regen the lock file

---

Updated